serviceregistry/src/main/java/eu/arrowhead/core/serviceregistry/database/service/ServiceRegistryDBService.java

```diff
@@ -64,6 +64,7 @@
 import eu.arrowhead.common.dto.shared.SystemResponseDTO;
```

Comment:
you changed address transformation toLowercase() to NetworkAddressPreProcessor.normalize() in multiple places. Unfortunately, the preprocessor does not always do the lowercase transformation (for example in case of domain names, simple strings).

Reply:
This was intentional. I'm just nut sure that the network address is case-insensitive in every environment (like Docker etc..) or in case of every possible protocol.